### DISCUSSION TITLE
Scaling down images quality

### DISCUSSION THREAD

**Discussion by saraOrkide** [pixijs/pixijs#9603] [General]
I load a photo in zoom 3, but the image quality is very low. In zoom 1, it is good, but when I zoom in higher, the image is not sharp.
The code I use is like this, I am working with version 7 of pixi and it is the latest version from pixi-viewport
const texture = await Assets.load(url);
            const image = Sprite.from(texture,
                { 
                    resolution: 1,
                    scaleMode: PIXI.SCALE_MODES.NEAREST,
                    resourceOptions: {
                        scale: viewport.scaled,
                        autoLoad: true,
                        MimeType: 'image/svg+xml'
                    }
                }
            );
            viewport.addChild(image);

**Comment by bigtimebuddy** [comment]
The scaleMode needs to be set on the Texture not on the sprite. Try this:
https://jsfiddle.net/75raybcu/

**Reply by saraOrkide** [reply]
This example does not work for svg file, for example, I replaced this file: https://upload.wikimedia.org/wikipedia/commons/8/84/Example.svg, but nothing was displayed.

**Reply by bigtimebuddy** [reply]
Wikipedia has CORS restrictions. Here's an example that works with SVG + scaleMode
https://jsfiddle.net/bigtimebuddy/e3dbsp12/

**Reply by saraOrkide** [reply]
It is not sharp
https://jsfiddle.net/bigtimebuddy/e3dbsp12/

**Reply by bigtimebuddy** [reply]
https://jsfiddle.net/bigtimebuddy/e3dbsp12/4/

You asked how to use scaleMode, not how to make it sharp.

**Reply by saraOrkide** [reply]
Yes, I apologize, I thought it would be sharp with scalemode, but I was wrong
In general, I want to show the svg in different zooms in sharp form

**Reply by bigtimebuddy** [reply]
`scaleMode` has two flavors: nearest and linear. When you scale the Sprite, this tells the GPU how to interpolate the pixels. Nearest looks pixelated and linear is smoother. Linear has limitations, especially if you are scaling down _a lot_ the edges can look not that sharp.

You may want to use canvas rendering (i.e., `drawImage`) directly and render multiple size variants to multiple canvases. Pixi does this under-the-hood with a single canvas. 2D canvas rendering can give the best possible visual result, even at extreme sizes.

**Comment by saraOrkide** [comment]
My photo svg
 with assets.load
 I loaded it, I want the quality of the photo not to decrease with zoom in and zoom out and it is always shown sharp

**Reply by antonfire** [reply]
Did you resolve this?
